Client/Application/TestAppDelegate.swift

```diff
@@ -113,7 +113,7 @@ class TestAppDelegate: AppDelegate {
 
         // Change to 0 to deactivate chron tabs
         if launchArguments.contains(LaunchArguments.ChronTabs) {
-            profile.prefs.setInt(0, forKey: PrefsKeys.ChronTabsPrefKey)
+            profile.prefs.setInt(0, forKey: PrefsKeys.FeatureFlags.ChronologicalTabs)
```

Comment:
Should this be `setInt` ? In BrowserViewController+URLBarDelegate.swift we're using `setBool` for that same pref.

Reply:
No idea. Haven't touched that at all. I'll take a look, but I'd bet the smarter idea is to keep it as is, given that it's currently working? hahahah